owncloudApp/src/main/java/com/owncloud/android/shares/viewmodel/OCShareViewModel.kt

```diff
@@ -74,11 +78,11 @@ class OCShareViewModel(
         expirationDateInMillis: Long,
         permissions: Int,
         publicUpload: Boolean
-    ): LiveData<Resource<List<OCShare>>> = shareRepository.updatePublicShareForFile(
+    ): LiveData<Resource<Void>> = shareRepository.updatePublicShareForFile(
```

Comment:
Why do you use Void? Shouldn't it be better use Unit?

Reply:
Done